server/core/src/irods_notify_service_manager.cpp

```diff
@@ -0,0 +1,99 @@
+#include "irods/irods_notify_service_manager.hpp"
+
+#include "irods/irods_logger.hpp"
+
+#include <cstddef>
+
+#ifdef IRODS_USE_LIBSYSTEMD
+
+#include <systemd/sd-daemon.h>
+
+namespace irods
+{
+    void notify_service_manager([[maybe_unused]] const std::size_t _msg_length,
+                                const char* _msg, // NOLINT(bugprone-easily-swappable-parameters)
```

Comment:
Please swap the order of these parameters.

Reply:
This is done intentionally to disambiguate it from the wrappers defined in the header.

Reply:
Done